### PR TITLE
Add usage information for the shim/monkey-patch

### PR DESCRIPTION
**Pull request by adamvoss**
Add usage information for the shim/monkey-patch to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,9 +7,18 @@ This package provides the built-in `util.promisify` in node v8.0.0 and later, an
...
+**Direct**
 ```js
 const promisify = require('util.promisify')
-// Use `promisify` just like the built-in
+// Use `promisify` just like the built-in method on `util`
+```
+
+**Shim**
+```js
+require('util.promisify').shim()
```
Comment (ljharb) [inline README.md]: `require('util.promisify/shim')()` would be even better.

Also, please add semicolons, throughout.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (adamvoss) [reply README.md]: Done.  I'll just assume it was a later decided to be a design mistake to expose shim on the main export.

Comment (ljharb) [reply README.md]: It wasn't a mistake; it's there for convenience - but it's still nicer to encourage people to use deep exports :-)

**Review by ljharb — APPROVED** [review]
Thanks!

I'll rebase this down to one commit prior to merging, if you don't get to it first.

**Comment by adamvoss** [comment]
I really wish I could rebase from my phone sometimes. You may beat me.

**Pull request merged**